documentation/assemblies/metrics/assembly_metrics-cruise-control.adoc

```diff
@@ -15,7 +15,7 @@ The example Grafana dashboard for Cruise Control provides:
 
 == Configuring Cruise Control
 
-You can enable the Cruise Control metrics in the `Kafka` resource using the `cruiseControl.metrics` property that contains the JMX exporter configuration about the metrics to expose.
+You can enable the Cruise Control metrics in the `Kafka` resource using the `cruiseControl.metricsConfig` property that contains a reference to the ConfigMap containing the JMX exporter configuration about the metrics to expose.
```

Comment:
```suggestion
Enable Cruise Control metrics using the `cruiseControl.metricsConfig` property in the `Kafka` resource to provide a reference to a ConfigMap that contains JMX exporter configuration about the metrics to expose.
```

Reply:
Would an example fragment from a ConfigMap help here?

Reply:
I guess this comment applies to all resources, not just cruise control. Is that correct?